edwards25519/Forc.toml

```diff
@@ -0,0 +1,7 @@
+[project]
+authors = ["HashCloak"]
+entry = "main.sw"
```

Comment:
Since we are now running the tests in Rust, this should instead point to `field_element.sw`